Source/WebKit/Platform/ExtraPrivateSymbolsForTAPI.h

```diff
@@ -34,6 +34,8 @@ void NetworkServiceInitializer();
 void WebContentServiceInitializer();
 void GPUServiceInitializer();
 
+void ExtensionEventHandler(xpc_connection_t);
```

Comment:
I think you should include `XPCSPI.h` here, just to be safe. As-is, it's probably relying on some other head getting imported first by tapi's frontend.

InstallAPI does use config.h (see `WebKit.xcconfig:228`), so SPI headers should be fine.

Reply:
Ah, that is a good point, will fix!

Thanks for reviewing!